ee/session_recordings/queries/test/test_session_recording_list_from_query.py

```diff
@@ -164,28 +159,6 @@ def test_effect_of_poe_settings_on_query_generated(
                 )
             self.assertQueryMatchesSnapshot(printed_query)
 
-    def _assert_is_pdi_filter(self, person_filtering_expr: list[Expr]) -> None:
-        assert person_filtering_expr[0].right.select_from.table.chain == ["person_distinct_ids"]
-        assert person_filtering_expr[0].right.where.left.chain == ["person", "properties", "rgInternal"]
-
-    def _assert_is_events_person_filter(self, person_filtering_expr: list[Expr]) -> None:
-        assert person_filtering_expr[0].right.select_from.table.chain == ["events"]
-        event_person_condition = [
-            x
-            for x in person_filtering_expr[0].right.where.exprs
-            if isinstance(x, CompareOperation) and x.left.chain == ["person", "properties", "rgInternal"]
-        ]
-        assert len(event_person_condition) == 1
-
-    def _matching_person_filter_expr_from(self, hogql_parsed_select: SelectQuery) -> list[Expr]:
-        where_conditions: list[Expr] = hogql_parsed_select.where.exprs
-        ands = [x for x in where_conditions if isinstance(x, And)]
-        assert len(ands) == 1
-        and_comparisons = [x for x in ands[0].exprs if isinstance(x, CompareOperation)]
-        assert len(and_comparisons) == 1
-        assert isinstance(and_comparisons[0].right, SelectQuery)
-        return and_comparisons
```

Comment:
these were a mess of mypy errors and after doing all this AST checking we then check the literal SQL string anyway
so we don't need this